FOR-DEVS.md

```diff
@@ -68,13 +68,20 @@ If you are part of theCourseForum engineering team, follow the instructions belo
     where `db_dump_file.sql` should be replaced with the latest DB file (e.g. `launch.sql`)
 7. Go to http://localhost:8000 in your browser and make sure everything works!
 
+### Windows setup (recommended)
+1. Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+2. Install Docker
+    - If you're using Windows 10 Pro, Enterprise, or Education, then follow [these instructions](https://docs.docker.com/docker-for-windows/install/) to install Docker Desktop, which comes with Docker and Docker Compose. A free copy of Windows 10 Education is available from UVA.
```

Comment:
Maybe add an instruction for how to check the OS?

Reply:
Done